charts/seed-bootstrap/charts/vpa/charts/runtime/templates/deployment-admission-controller.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     app: vpa-admission-controller
-{{ toYaml .Values.labels | indent 4 }}
+{{ toYaml .Values.deploymentLabels | indent 4 }}
```

Comment:
Shouldn't these labels get merged together with `.Values.labels`? If I see it correctly, the deployment will now only be labelled with the `role=controlplane` labels while it had earlier

```yaml
app: vpa-admission-controller
garden.sapcloud.io/role: vpa
```

Reply:
Good point. My thought of replacing it was the following:

Previously:
```
app: vpa-admission-controller
garden.sapcloud.io/role: vpa
```

With this PR:
```
app: vpa-admission-controller
gardener.cloud/role: seed
```

If we merged the label then we would have this result which is somehow confusing because `garden.sapcloud.io/role` is the old label key with a different value as the one for `gardener.cloud/role`.
```
app: vpa-admission-controller
garden.sapcloud.io/role: vpa
gardener.cloud/role: seed
```

I thought, replacing the label is not really critical because we haven't used any selectors for `garden.sapcloud.io/role: vpa` so far. What do you think?

Reply:
Sure, I don't mind changing the labels if we are sure that we don't break other functionality that depend on these labels to be present. I remember we had a label selector for cleaning up the VPA earlier when it was backed by a feature gate, but I don't find this code anymore, so probably you're right and it's fine. Thank you.